### PR TITLE
Adding style to sign up forms

### PR DESCRIPTION
**Pull request by shreyabarua**
Add css to make sign up forms look more form-like
Add redirect from sign up form pages to resume review page after they click submit.

### CODE REVIEW

File: resume-buddy/src/main/java/com/google/sps/servlets/ReviewerDataServlet.java [hunk 1]
```diff
@@ -40,14 +40,12 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
...
-    /* TODO: add a response.sendRedirect() to wherever the user should
-     * be navigated to next (either homepage or resume review page)
-     */ 
+    response.sendRedirect("resume-review.html");
   }
 
   /**
    * @return the request parameter, or the default value if the parameter was not specified by the
-   * client
+   *     client
```
Comment (oydang) [inline resume-buddy/src/main/java/com/google/sps/servlets/ReviewerDataServlet.java]: Nit: auto format must have added this extra space again before `client`... strange

File: resume-buddy/src/main/java/com/google/sps/servlets/RevieweeDataServlet.java [hunk 2]
```diff
@@ -40,14 +40,12 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
     datastore.put(revieweeEntity);
 
-    /* TODO: add a response.sendRedirect() to wherever the user should
-     * be navigated to next (either homepage or resume review page)
-     */ 
+    response.sendRedirect("resume-review.html");
```
Comment (oydang) [inline resume-buddy/src/main/java/com/google/sps/servlets/RevieweeDataServlet.java:43]: Do the reviewer and reviewee get some kind of notification that their form was successfully submitted or do they just get directed back to `resume-review.html`?

File: resume-buddy/src/main/webapp/reviewee-form.html [hunk 3]
```diff
@@ -6,63 +6,67 @@
...
+        <input type="text"><br><br>
+        <label>School year:</label><br>
+        <select>
+        <option disabled selected>Choose one</option>
+        <option>High school</option>
+        <option>First year</option>
+        <option>Second year</option>
+        <option>Third year</option>
+        <option>Fourth year</option>
+        <option>Post grad</option>
```
Comment (oydang) [inline resume-buddy/src/main/webapp/reviewee-form.html]: Nit: shouldn't this section be indented? Try to auto-format this whole file with 2-space indentations.

File: resume-buddy/src/main/webapp/reviewee-form.html [hunk 4]
```diff
@@ -6,63 +6,67 @@
...
+        <h3 class = "heading">Reviewer Preferences</h3>
+        <label>Field of Work:</label><br>
+        <select>
+        <option disabled selected>Choose one</option>
+        <option>Computer Science</option>
+        <option>Healthcare/Medicine</option>
+        <option>Education</option>
+        <option>Engineering</option>
+        <option>Business</option>
+        <option>Other</option>
```
Comment (oydang) [inline resume-buddy/src/main/webapp/reviewee-form.html]: Nit: this section should also be indented?

File: resume-buddy/src/main/webapp/reviewee-form.html [hunk 5]
```diff
@@ -6,63 +6,67 @@
...
+        <br>
+        <label>Other:</label><br>
+        <input type="text"><br><br>
+        <label>Highest level of education:</label><br>
+        <select>
+        <option disabled selected>Choose one</option>
+        <option>Bachelor's degree</option>
+        <option>Master's degree</option>
+        <option>Doctorate</option>
+        <option>Doesn't matter</option>
```
Comment (oydang) [inline resume-buddy/src/main/webapp/reviewee-form.html]: Nit: should be intended

File: resume-buddy/src/main/java/com/google/sps/servlets/RevieweeDataServlet.java [hunk 6]
```diff
@@ -40,14 +40,12 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
...
   }
 
   /**
    * @return the request parameter, or the default value if the parameter was not specified by the
-   * client
+   *     client
```
Comment (oydang) [inline resume-buddy/src/main/java/com/google/sps/servlets/RevieweeDataServlet.java]: Nit: auto format must have added this extra space again before client... strange

**Review by oydang — CHANGES_REQUESTED** [review]

File: resume-buddy/src/main/webapp/reviewee-form.html [hunk 7]
```diff
@@ -6,63 +6,67 @@
...
+        <label>Doesn't matter</label><br>
+        <br>
+        <p class = "heading">Upload Resume PDF</p>
+        <input type="file"><br>
+        <br>
+        <input type="submit">
+    </form>
+  </div>
+</body>
 </html>
```
Comment (AniMachaidze) [inline resume-buddy/src/main/webapp/reviewee-form.html]: Missing empty line at the end of the file

File: resume-buddy/src/main/webapp/reviewer-form.html [hunk 8]
```diff
@@ -6,54 +6,58 @@
...
+        <option>5 - 10</option>
+        <option>greater than 10</option>
+        </select>
+        <br><br>  
+        <br>
+        <input type="submit" value="Submit">
+    </form>
+  </div>
+</body>
 </html>
```
Comment (AniMachaidze) [inline resume-buddy/src/main/webapp/reviewer-form.html]: Missing empty line at the end of the file

File: resume-buddy/src/main/java/com/google/sps/servlets/RevieweeDataServlet.java (hunk 2 above)
Comment (shreyabarua) [reply resume-buddy/src/main/java/com/google/sps/servlets/RevieweeDataServlet.java:43]: for now they just get redirected but that's a good idea I'll add that in another PR

**Review by AniMachaidze — CHANGES_REQUESTED** [review]

**Review by AniMachaidze — APPROVED** [review]

**Review by oydang — APPROVED** [review]

**Pull request merged**